### DISCUSSION TITLE
Screen Blanking out

### DISCUSSION THREAD

**Discussion by seanas3** [Stellarium/stellarium#2160] [General]
As any one noticed that when they use Stellarium, the Monitor will occasionally blank out and then come back. It acts as if it's trying to pout my monitors to sleep.

**Comment by gzotti** [comment]
No. This may depend on computer type, hardware capabilities, operating system, various drivers, monitor cable, power supply, or even a faulty monitor.

**Reply by seanas3** [reply]
I thought about that, but this is the only application that causes my "monitors" to behave this way.

**Comment by Keatah** [comment]
Yes. Seen that several times already. Only with Stellarium. Everything else works fine.
